future-apps/actor/contracts/test/mocks/DestinationMock.sol

```diff
@@ -2,9 +2,16 @@ pragma solidity 0.4.24;
 
 
 contract DestinationMock {
-    uint256 test;
+    bool public expensiveFallback;
+    uint256 public counter;
+
+    constructor(bool _expensiveFallback) {
```

Comment:
Although it's just a test, it wouldn't hurt to add `public` here to avoid a warning.